### PR TITLE
Add release notes for version 0.84.0

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [ ] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -10,6 +10,42 @@ toc_max_heading_level: 2
...
 
+## March 2022
+
+### Version 0.84.0
+
+#### Additions
+
+##### Semgrep CLI lists supported languages
+
+Semgrep CLI now includes `--show-supported-languages` flag to display the list of languages supported by semgrep. Thanks to John Wu for this contribution! ([#4754](https://github.com/returntocorp/semgrep/pull/4754))
```
Comment (s-santillan) [inline docs/release-notes.md:21]: In "... display the list of languages supported by semgrep"

We should capitalize Semgrep unless we're referring to the command line  `semgrep` in which case use backticks.

File: docs/release-notes.md [hunk 2]
```diff
@@ -10,6 +10,42 @@ toc_max_heading_level: 2
...
+- Semgrep scans for JSX self closing tags (XML elements) such as `<foo />` can result in a match of explicitly closed tags, for example: `<foo >some child</foo>`. You can now disable this behavior by rule options: `xml_singleton_loose_matching: false` (#4730)
+- New rule option `xml_attrs_implicit_ellipsis` that allows you to disable the implicit ellipsis `...` that was added to JSX attributes patterns.
+
+##### Updated validation of rules
+
+The `semgrep --config [file] --validate` now checks for invalid metavariables.
+
+##### The `project-depends-on` now supports more languages
+
+You can now use `r2c-internal-project-depends-on` with lockfiles for Java, Go, Ruby, and Rust. ([#4699](https://github.com/returntocorp/semgrep/pull/4699))
```
Comment (s-santillan) [inline docs/release-notes.md:36]: This is a pretty big deal. Just FYI, no need to do anything.

**Review by s-santillan — APPROVED** [review]
Some non-blocking nit suggestions. Because they're non-blocking, I am approving and will leave to you which suggestions you'd like to take.

File: docs/release-notes.md (hunk 1 above)
Comment (adamkvitek) [reply docs/release-notes.md:21]: Oh yes, of course... how could it slip through... thank you. :D

File: docs/release-notes.md [hunk 3]
```diff
@@ -10,6 +10,42 @@ toc_max_heading_level: 2
...
+##### JSX (JavaScript) improvement
+
+Semgrep CLI now provides the following improvements for JSX (JavaScript extension) scans:
+
+- Semgrep scans for JSX self closing tags (XML elements) such as `<foo />` can result in a match of explicitly closed tags, for example: `<foo >some child</foo>`. You can now disable this behavior by rule options: `xml_singleton_loose_matching: false` (#4730)
+- New rule option `xml_attrs_implicit_ellipsis` that allows you to disable the implicit ellipsis `...` that was added to JSX attributes patterns.
+
+##### Updated validation of rules
+
+The `semgrep --config [file] --validate` now checks for invalid metavariables.
```
[the remark this answers is not in the document]
Comment (adamkvitek) [reply docs/release-notes.md:32]: Maybe you will convince me about the opposite, but I am generally not sure in this case. Some TWs I have seen were trying to put definite articles in order to somehow show that this is a noun (basically the whole inline code is a noun). But I do not have a strong basis for it in the Google style guide. See: https://developers.google.com/style/code-in-text?hl=en (the example with "The `ADDRESS` constant's value is defined in the `settings.h` file." @s-santillan

File: docs/release-notes.md (hunk 2 above)
Comment (adamkvitek) [reply docs/release-notes.md:36]: Maybe you can tell me more later, please. Just so I know what is happening.

**Pull request merged**